documentation/dashboard/working_with_data.md

```diff
@@ -118,6 +122,30 @@ Observations must be defined before they can be added to an aggregate (see
 
    <img class="my-3" src="/images/aggregate_observation_form.png"/>
 
+### Examples
+
+We would like to create an aggregate from three observations. Observations 1 and 2 have
+data starting on 2020-01-01T00:00Z.  Observation 3 has data starting on
+2020-03-01T00:00Z. There are two ways in which we can create this aggregate.
+
+1. **Recommended approach**: Add observations with different *effective from* parameters. We add 
+   observations 1 and 2 with an *effective from*  of
+   2020-01-01T00:00Z. Then we add observation 3 with an
+   *effective from* of 2020-03-01T00:00Z. With this configuration, the computed
+   aggregate from 2020-01-01T00:00Z to 2020-03-01T00:00Z will contain
+   observations 1 and 2, and the computed aggregate after 2020-03-01T00:00Z
+   will contain observations 1, 2, and 3.
+
+2. Alternative approach: Add all observations with the same *effective from*,
+   accept NaN values for period with missing data. Using the [process above](#add-observations-to-an-aggregate),
+   we add observations 1, 2, and 3 with an *effective from* of
+   2020-01-01T00:00Z. The Solar Forecast Arbiter cannot safely assume a value
+   for the missing data, so it will compute the aggregate as NaN from
+   2020-01-01T00:00Z to 2020-03-01T00:00Z. The computed aggregate after
+   2020-03-01T00:00Z will contain observation 1, 2, and 3, as expected. If the
+   aggregate represents a sum of power, we may choose to upload 0 values for
+   observation 3 for the missing time period. This choice should be
+   communicated to other users of the data.
 
 ### Remove an Observation from an Aggregate
 {: .anchor}
```

Comment:
commenting here since it's as close as I can get...

> Aggregate values will only contain data from constituent observations between
their *Effective From* and *Effective Until* dates.

this sentence should be in "create new aggregate". 

We should rename this section to something like "End an Observation's contribution to an Aggregate". Remove is too similar to Delete for my taste.